policies/terms_of_membership.md

```diff
@@ -0,0 +1,107 @@
+Terms of Membership
+
+# Charitable Purpose
+
+The Charity’s objects (“Objects”) are the education of the public in STEAM (Science, Technology, Engineering, Arts and Mathematics) and the provision of community accessible workspaces to facilitate STEAM outreach and education in Northern Ireland, primarily focused in the Belfast area
+
+See [here](articles_of_association.md) for more information.
```

Comment:
change "here" to "the articles of association" (more descriptive & better for screen readers)